plugins/nitrokey/README.md

```diff
@@ -0,0 +1,9 @@
+Nitrokey Support
+================
+
+Introduction
+------------
+
+This plugin is used to get the correct version number on Nitrokey storage
+devices. These devices have updatable firmware but so far no updates are
+available from the vendor.
```

Comment:
So then will this actually show up as two distinct USB devices in fwupd?  One for the ```Atmel Corp. at32uc3a3 DFU bootloader``` and one for the ```Clay Logic```?  I'm wondering if it's worth drawing a connection between the two in some way if so.

Reply:
I've not tested this yet, but I think with the right DFU quirk the bootloader device will show up only after the runtime has disappeared, and I *think* they'll have the same device-id and gnome-software will magically work. It certainly needs testing.

Reply:
I suppose before merging you'll want to make sure that the device isn't getting stuck into bootloader mode as a result of this interaction (and not usable by the OS), but otherwise that sounds great!

Reply:
I'm leaving the actual switch to bootloader out of this commit; it needs a few architectural changes in the daemon to do correctly. I'm working on those changes, but it's taking a bit of time to get my head around all the subtleties of switching plugin mid-upgrade.

Reply:
OK.